emuu-client/src/components/NavbarPostLogin/HeaderPostLogin.js

```diff
@@ -32,14 +33,20 @@ function HeaderPostLogin() {
     <>
       {[false].map((expand) => (
         <Navbar
-                  key={expand}
-                  bg="dark"
-                  variant="dark"
-                  expand={expand}
-                  className="mb-3"
-                >
+          key={expand}
+          bg="dark"
+          variant="dark"
+          expand={expand}
+          className="mb-3"
+        >
           <Container fluid>
-            <Navbar.Brand href="/Home">EMUU</Navbar.Brand>
+            <Navbar.Brand href="/">
+              <img
+                src="https://firebasestorage.googleapis.com/v0/b/emuu-1ee85.appspot.com/o/images%2FPicture2.png?alt=media&token=c2e0afa7-7f0b-4b75-b9f9-e38043b729bb"
```

Comment:
It would be better if you had a seperate file for the logo, instead of having a url here for it